eng/pipelines/pr.job.yml

```diff
@@ -78,7 +78,7 @@ jobs:
     condition: succeeded()
     inputs:
       command: test
-      arguments: --no-restore --restore:false --no-build --framework $(TestTargetFramework) --blame --logger trx --binarylogger:"$(BinlogDirectory)02-RunTests.binlog"
```

Comment:
> Finally, I've disabled the crash dump helper when running tests since all of the memory dumps it provided were always for the main vstest.console.exe which wasn't helpful since the tests themselves are running in a different process

Have you talked to the team who owns `dotnet test`?  This sounds very surprising so sounds like something that should be fixed for all customers, or if we're doing something wrong, they can help us.

Reply:
I see a few issues in the vstest repo:
https://github.com/microsoft/vstest/issues/3936
https://github.com/microsoft/vstest/issues/2593
https://github.com/microsoft/vstest/issues/4376
https://github.com/microsoft/vstest/issues/2952

Nothing specific to the problem I was seeing.  Supposedly the blame stuff also doesn't supply enough command-line arguments to capture other exceptions than `StackOverflowException` and `AccessViolationException`

https://github.com/microsoft/vstest/blob/main/src/Microsoft.TestPlatform.Extensions.BlameDataCollector/ProcDumpArgsBuilder.cs#L33

Although now there's an env var we can set to include more arguments to proc dump.  The main point in all of this is it seems like adding `--diag` to the `vstest.console.exe` execution is better.  I don't think we have that many "test crashes".  But if we feel like we need the memory dumps back, we can do so at a later time in my opinion.